lib/in_parallel.rb

```diff
@@ -152,7 +152,7 @@ def self.wait_for_processes(proxy = self, binding = nil, timeout = nil, kill_all
                 end
                 results_map[process_info[:index]] = { process_info[:tmp_result] => marshalled_result }
               ensure
-                File.delete(process_info[:std_out]) if File.exists?(process_info[:std_out])
+                File.delete(process_info[:std_out]) if File.exist?(process_info[:std_out])
```

Comment:
The atomic `FileUtils.rm_f` is actually a better way, but I'm not sure how many changes you want to make here.